client/post-editor/editor-sidebar/header.jsx

```diff
@@ -54,9 +54,8 @@ function EditorSidebarHeader( { translate, type, siteId, showDrafts, toggleDraft
 				</Button>
 			) }
 			{ type === 'post' && siteId && (
-				<PostCountsData siteId={ siteId } status="draft">
-					<DraftsButton onClick={ toggleDrafts } />
-				</PostCountsData>
+				<QueryPostCounts siteId={ siteId } type="post" /> ) && (
```

Comment:
This won't work to add the two components to the page. Probably why I'm not seeing count in my editor.

Instead, you could try:

```jsx
{ type === 'post' && siteId && [
	<QueryPostCounts key="query-count" siteId={ siteId } type="post" />,
	<DraftsButton key="draft-button" onClick={ toggleDrafts } />
] }
```

Or:

```jsx
{ 'post' === type && sideId && <QueryPostCounts siteId={ siteId } type="post" /> }
{ 'post' === type && <DraftsButton onClick={ toggleDrafts } /> }
```